python/ray/_private/usage/usage_lib.py

```diff
@@ -787,6 +797,19 @@ def get_cluster_metadata(gcs_client) -> dict:
     )
 
 
+def generate_disabled_report_data() -> UsageStatsToReport:
+    """Generate the report data indicating usage stats is disabled"""
+    data = UsageStatsToReport(
+        schema_version=usage_constant.SCHEMA_VERSION,
+        source=os.getenv(
+            usage_constant.USAGE_STATS_SOURCE_ENV_VAR,
+            usage_constant.USAGE_STATS_SOURCE_OSS,
+        ),
+        collect_timestamp_ms=int(time.time() * 1000),
```

Comment:
Those are the only data we send to the server when usage stats is disabled.

Reply:
so only the timestamp?

Reply:
cc @richardliaw @robertnishihara This is probably what you care about.